internal/controller/policyserver_controller_configmap.go

```diff
@@ -22,13 +22,68 @@ import (
 
 const dataType string = "Data" // only data type is supported
 
+type policyGroupMember struct {
+	URL                   string                            `json:"url"`
+	Settings              runtime.RawExtension              `json:"settings,omitempty"`
+	ContextAwareResources []policiesv1.ContextAwareResource `json:"contextAwareResources,omitempty"`
+}
+
 type policyServerConfigEntry struct {
 	NamespacedName        types.NamespacedName              `json:"namespacedName"`
-	URL                   string                            `json:"url"`
+	URL                   string                            `json:"url,omitempty"`
```

Comment:
if we aren't using a different structure to marshall both Groups and Policies, and we want to have omitempty in URL and AllowedToMutate, I would prefer to check on Unmarshalling and:
- error if the json is a normal Policy (`len(Policies) == 0`) and doesn't provide `URL` or `AllowedToMutate`.
- error if the json is a Group (`len(Policies) != 0) and provides a `URL` or AllowedToMutate`.